draft-gpew-priv-ppm.md

```diff
@@ -165,12 +162,17 @@ Leader:
 : A distinguished aggregator that coordinates input validation and data
    collection.
 
-Output:
+Aggregate result:
 : The output of the aggregation function over a given set of reports.
 
+Aggregate share:
+: A share of the aggregate result emitted by an aggregator. Aggregate shares are
+   reassembled by the collector into the final output.
+
 Output share:
-: The share of an output emitted by an aggregator. Output shares can be
-   reassembled by the leader into the final output.
+: An aggregator's share of the output of the VDAF
+   {{?I-D.draft-cfrg-patton-vdaf}} preparation step. Many output shares are
+   combined into an aggregate share via the VDAF aggregation algorithm.
```

Comment:
This may be the VDAF term, but it seems kind of unfortunate. Why  isn't this called "input"?

Reply:
As discussed offline, the "prepare step" is the MPC that maps input shares to output shares (using the aggregation parameter to control the output).